.mci.yml

```diff
@@ -88,6 +88,7 @@ functions:
                  rm -rf "mongo-cxx-driver"
                  rm -fr "mongo-c-driver"
                  rm -fr mongod
+                 rm -fr drivers-evergreen-tools
```

Comment:
I'm not sure why Windows variants only started failing now, but I saw a related failure here:

https://evergreen.mongodb.com/task/cxx_driver_windows_2k8_release_compile_and_test_with_shared_libs_060b6dd5fc23eade27710a30e6a12eeecb84e38a_20_06_02_15_58_04

Reply:
Yes, I've seen the failure for when the directory already exists.  Seems worthwhile to add this here!  On one of my branches, I've also changed the line where we clone drivers-evergreen-tools to check if the directory exists first.  With this cleanup added, that check is probably overkill.